src/base58.cpp

```diff
@@ -188,9 +188,6 @@ bool CBase58Data::SetString(const std::string& str)
 
 std::string CBase58Data::ToString() const
 {
-//    if(vchData.To == "0x202020")
-//      return "FUND_CONTRIBUTION";
-
```

Comment:
why don't we need this anymore?

Reply:
its a ghost comment from some internal test i did i forgot to remove. we never needed it.